src/apis/vector.ts

```
@@ -297,9 +297,24 @@ export default class VectorAPI implements VectorStorage {
 
 			// Execute the operation within the new context
 			return await context.with(spanContext, async () => {
-				const resp = await DELETE<VectorDeleteResponse>(
-					`/vector/2025-03-17/${encodeURIComponent(name)}/${encodeURIComponent(key)}`
-				);
+				if (keys.length === 0) {
+					span.setStatus({ code: SpanStatusCode.OK });
+					return 0;
+				}
+
+				let resp;
```

Biome (1.9.4)

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)